neuralcoref/train/conllparser.py

```diff
@@ -348,11 +348,11 @@ def add_conll_utterance(
         # Convert conll tokens coref index in spacy tokens indexes
         identified_gold = [False] * len(corefs)
         for coref in corefs:
-            assert (
-                coref["label"] is not None
-                and coref["start"] is not None
-                and coref["end"] is not None
-            ), ("Error in coreference " + coref + " in " + parsed)
+            missing_values = [key for key in ['label', 'start', 'end', ] if coref[key] is None]
```

Comment:
perhaps even more generic would be `if coref.get(key, None) is None`. That includes also cases where the key is not present (which shouldn't really happen)

Reply:
```suggestion
            missing_values = [key for key in ['label', 'start', 'end', ] if coref.get(key, None) is None]
```